source/pymongo-async-migration.txt

```diff
@@ -0,0 +1,357 @@
+.. _pymongo-async-migration:
+
+===================================
+Migrate to the {+driver-async+} Driver
+===================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: motor, async, refactor, migration
+
+Overview
+--------
+
+The {+driver-async+} driver is a unification of {+driver-short+} and the `Motor
+library <https://www.mongodb.com/docs/drivers/motor/>`__. In this guide, you can
+identify the changes you must make to migrate an application from
+{+driver-short+} or Motor to the {+driver-async+} driver.
+
+Migrate From {+driver-short+}
+--------------------
+
+The {+driver-async+} driver behaves similarly to {+driver-short+}, but
+all methods that perform network operations are coroutines and must be awaited.
+To migrate from {+driver-short+} to {+driver-async+}, you must update your code
+in the following ways:
+
+- Replace all uses of ``MongoClient`` with ``AsyncMongoClient``.
+- Add the ``await`` keyword to all asynchronous method calls.
+- If an asynchronous method is called within a function, mark the function as ``async``.
+
+The following sections describe how to implement the asynchronous API.
```

Comment:
I'd also link to the official [Python async docs here](https://docs.python.org/3/library/asyncio.html) for some additional context.